goodtablesio/integrations/github/blueprint.py

```diff
@@ -265,3 +222,43 @@ def api_repo(repo_id):
         'repo': repo,
         'error': error,
     }), code)
+
+
+@github.route('/api/repo/<repo_id>/activate')
+@login_required
+def api_repo_activate(repo_id):
+    error = None
+    token = current_user.github_oauth_token
+    if not token:
+        error = 'No valid GitHub token found'
```

Comment:
No need to continue the execution if there is no token right? just return the error